modules/polling/helpers/recentlyUsedGaslessVotingCheck.ts

```diff
@@ -0,0 +1,14 @@
+import { getRecentlyUsedGaslessVotingKey } from 'modules/cache/constants/cache-keys';
+import { cacheGet } from 'modules/cache/cache';
+import { SupportedNetworks } from 'modules/web3/constants/networks';
+
+export async function recentlyUsedGaslessVotingCheck(
+  voter: string,
+  network: SupportedNetworks
+): Promise<boolean> {
+  const cacheKey = getRecentlyUsedGaslessVotingKey(voter);
+
+  const recentlyUsedGaslessVoting = await cacheGet(cacheKey, network);
```

Comment:
does this work on the client side ? :/ im confused

Reply:
if this works on the client side it means the cacheGet and env variable are exposed, it shouldn't be. 

this might need to be an API call to api/polling/vote?address=0x213 (GET call with some address parameters)

Reply:
Haven't been able to verify cause gasless isn't working for me locally yet, but no, don't think it would work. `cacheGet` will return null since `config.USE_CACHE` wouldn't be accessible client-side. Will have to refactor to an endpoint.